### DISCUSSION TITLE
enh: support prompt generation

### DISCUSSION THREAD

**Discussion by bdqfork** [open-webui/open-webui#6213] [General]
# Feature Request

support prompt generation via openai meta prompt https://platform.openai.com/docs/guides/prompt-generation

**Comment by flefevre** [comment]
Personnaly I have created a meta model assistant directl in open webui in charge of helping user to create their own model assistant with custom prompt' they interact with it in the chat UI of open webui and then open a second tab in the web browser to fill the form of open webui model creator. ( We could not anymore copy paste the full JSON since the JSON model overview is locked)
In addition to 'prompt generation' it could be great to be able to manage their life cycle. I have tried to describe the idea here https://github.com/open-webui/open-webui/discussions/5915

Connecting Langfuse with their prompt management to open web ui or creating a dedicated zone in openwebui to store 'prompt ' with their version and with their access management. A team could have or not access to prompt. Those prompt could be exposed in the user interface and in a future fastapi of openwebui?

Some ideas to help keeping open web ui the best tools for llm
